frontend/packages/integration-tests-cypress/views/yaml-editor.ts

```diff
@@ -13,5 +13,13 @@ export const setEditorContent = (text: string) => {
 // initially yamlEditor loads with all grey text, finished loading when editor is color coded
 // class='mtk26' is the light blue color of property such as 'apiVersion'
 export const isLoaded = () => cy.get("[class='mtk26']").should('exist');
-export const clickSaveCreateButton = () => cy.byTestID('save-changes').click();
+// since yaml editor class mtk26 is a font class it doesn't work on an import page with no text
+// adding a check for the 1st line number, AND providing a wait allowed the load of the full component
+export const isImportLoaded = () => {
+  // eslint-disable-next-line cypress/no-unnecessary-waiting
+  cy.wait(2000);
+  cy.get("[class='line-numbers']").should('exist');
+};
+export const clickSaveCreateButton = () => cy.byTestID('save-changes').click({ force: true });
```

Comment:
Why is `force: true` required?

Reply:
My understanding the use of `force: true` in these calls is for when the UI viewport doesn't have the item directly visible but it's still in the DOM; ie scrolled down further or via some hidden aspect/overlay -- this part concerns me a bit as I wonder if background renderings (other tabs for instance) would be caught by this.

Reply:
We may need to dig into this, I've noticed in the past the [Create] button for the YAML editor sometimes runs into this.  I thought it might have something to do with the hidden "View [schema] sidebar", but Import YAML editor doesn't have that sidebar.

Reply:
@spadgett @andrewballantyne @dtaylor113 i was able to remove the force click for these tests.